src/lib.rs

```diff
@@ -133,6 +133,18 @@ impl Cifs {
         Ok(data)
     }
 
+    pub async fn delete(&mut self, share: &Share, path: &str) -> Result<(), Error> {
+        let filename = path.replace("/", "\\");
```

Comment:
We could think about extracting this into a shared `sanitizePath` function?

Reply:
good idea